### PR TITLE
Require dismissal of archive operations

### PR DESCRIPTION
**Pull request by WilliamBZA**
Added dismissal code @WojcikMike.

Check if you're happy and you and merge into your branch

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Also added an extra state

**Comment by WilliamBZA** [comment]
Also fixed the status check in the archive in progress function

File: src/ServicePulse.Host/app/js/services/services.service-control.js [hunk 1]
```diff
@@ -221,6 +221,13 @@
                 });
         }
 
+        function acknowledgeArchiveGroup(groupId, successText, failureText) {
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/services/services.service-control.js]: from what I see successText and failureText is not being used in this code. Please remove those parameters

File: src/ServicePulse.Host/app/js/views/failed_groups/controller.js [hunk 2]
```diff
@@ -123,8 +123,8 @@
             return group.workflow_state.status !== 'none' && (group.workflow_state.status !== 'completed' || group.need_user_acknowledgement === true) && !vm.isBeingArchived(group.workflow_state.status);
         };
 
-        vm.isBeingArchived = function (status) {
-            return status === "archivestarted" || status === "archiveprogressing" || status === "archivefinalizing" || status === "archivecompleted";
+        vm.isBeingArchived = function (group) {
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/views/failed_groups/controller.js]: This fix require some more work. 
 * There are at least 2-3 places where we still call isBeingArchived passing status not a  group (easy fix)
 * `operation_status` that you check contains status that is not to lower, so either tha names need to be adjusted or tolower used (easy fix)
 * `operation_status` is not being updated by our event handlers. So only when the list is refreshed the state will be actual. So we should update operation_status field whenever an event is handled.

**Comment by WilliamBZA** [comment]
Changes made @WojcikMike

File: src/ServicePulse.Host/app/js/views/failed_groups/controller.js [hunk 3]
```diff
@@ -124,7 +122,7 @@
         };
 
         vm.isBeingArchived = function (group) {
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/views/failed_groups/controller.js]: The parameter is named group, but you use status in the method ;)

Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/failed_groups/controller.js]: You wrote the method, I'm just using it ;)

Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/failed_groups/controller.js]: Nevermind, I'm dumb haha

**Pull request merged**